CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### v6.5.0 (2022-03-16)
```

Comment:
Just a heads-up this date is now way off.

Reply:
le sigh. ill update